src/types/response.rs

```diff
@@ -68,9 +68,11 @@ impl Response {
 impl ToPyObject for Response {
     fn to_object(&self, py: Python) -> PyObject {
         let headers = self.headers.clone().into_py(py).extract(py).unwrap();
-        let description = String::from_utf8(self.description.to_vec())
-            .unwrap()
-            .to_object(py);
+        let description = match String::from_utf8(self.description.to_vec()) {
```

Comment:
```suggestion
        // The description should only be either string or binary.
        // it should raise an exception otherwise
        let description = match String::from_utf8(self.description.to_vec()) {
```